rc/gdscript.kak

```diff
@@ -0,0 +1,110 @@
+# http://godotengine.org
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .*[.](gd) %{
+    set-option buffer filetype gdscript
+}
+
+# Initialization
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+hook global WinSetOption filetype=gdscript %{
+    require-module gdscript
+
+    set-option window static_words %opt{gdscript_static_words}
+
+    hook window InsertChar \n -group gdscript-insert gdscript-insert-on-new-line
+    hook window InsertChar \n -group gdscript-indent gdscript-indent-on-new-line
+    # cleanup trailing whitespaces on current line insert end
+    hook window ModeChange pop:insert:.* -group gdscript-trim-indent %{ try %{ execute-keys -draft <semicolon> <a-x> s ^\h+$ <ret> d } }
+    hook -once -always window WinSetOption filetype=.* %{ remove-hooks window gdscript-.+ }
+}
+
+hook -group gdscript-highlight global WinSetOption filetype=gdscript %{
+    add-highlighter window/gdscript ref gdscript
+    hook -once -always window WinSetOption filetype=.* %{ remove-highlighter window/gdscript }
+}
+
+provide-module gdscript %§
+
+# Highlighters
+# ‾‾‾‾‾‾‾‾‾‾‾‾
+
+add-highlighter shared/gdscript regions
+add-highlighter shared/gdscript/code default-region group
+add-highlighter shared/gdscript/double_string region '"'   (?<!\\)(\\\\)*"  fill string
+add-highlighter shared/gdscript/single_string region "'"   (?<!\\)(\\\\)*'  fill string
+add-highlighter shared/gdscript/comment       region '#'   '$'              fill comment
+
+# integers
+add-highlighter shared/gdscript/code/ regex '(?i)\b0b[01]+l?\b'               0:value
+add-highlighter shared/gdscript/code/ regex '(?i)\b0x[\da-f]+l?\b'            0:value
+add-highlighter shared/gdscript/code/ regex '(?i)\b0o?[0-7]+l?\b'             0:value
+add-highlighter shared/gdscript/code/ regex '(?i)\b([1-9]\d*|0)l?\b'          0:value
+# floats
+add-highlighter shared/gdscript/code/ regex '\b\d+[eE][+-]?\d+\b'             0:value
+add-highlighter shared/gdscript/code/ regex '(\b\d+)?\.\d+\b'                 0:value
+add-highlighter shared/gdscript/code/ regex '\b\d+\.'                         0:value
+# functions
+add-highlighter shared/gdscript/code/ regex _?[a-zA-Z]\w*\s*(?=\()            0:function
```

Comment:
Could the actual function name be captured here, to avoid highlighting trailing whitespace (the `\s*`)?